.travis.yml

```diff
@@ -14,4 +14,13 @@ script:
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "virtualenv /tmp && test -f /tmp/bin/pip"
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "virtualenv -ppython3.5 /tmp && test -f /tmp/bin/pip"
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "python3.6 -m venv /tmp && test -f /tmp/bin/pip"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "virtualenv-2.7 /tmp && /tmp/bin/pip -vvv install cryptography" 
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "python3.5 -m venv /tmp && /tmp/bin/pip -vvv install cryptography" 
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "python3.6 -m venv /tmp && /tmp/bin/pip -vvv install cryptography" 
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "docker-compose --version"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "printf '[tox]\\nskipsdist=true\\n[testenv]\\ndeps=pytest\\ncommands=py.test --version\\n' > tox.ini && pip install tox && tox -e py27"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "printf '[tox]\\nskipsdist=true\\n[testenv]\\ndeps=pytest\\ncommands=py.test --version\\n' > tox.ini && pip install tox && tox -e py35"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "printf '[tox]\\nskipsdist=true\\n[testenv]\\ndeps=pytest\\ncommands=py.test --version\\n' > tox.ini && pip install tox && tox -e py36"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "printf '[tox]\\nskipsdist=true\\n[testenv]\\ndeps=pytest\\ncommands=py.test --version\\n' > tox.ini && pip3 install tox && tox -e py27"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "printf '[tox]\\nskipsdist=true\\n[testenv]\\ndeps=pytest\\ncommands=py.test --version\\n' > tox.ini && pip3 install tox && tox -e py35"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "printf '[tox]\\nskipsdist=true\\n[testenv]\\ndeps=pytest\\ncommands=py.test --version\\n' > tox.ini && pip3 install tox && tox -e py36"
```

Comment:
Not sure what the last 6 lines are testing. I'm assuming they are testing if pytest can be run with different python versions, but I'm unsure why you have both pip as well as pip3.

Are you only testing python2 pip vs python3 pip? If not, I would expect pip with python 2.7, 3.5 and 3.6 (but that would mean 9 tests instead of 6, and it would get even more fun if we will more versions in the future, because it is quadratic).

Reply:
this is testing some cross install. between different versions. clearly not all. but at least the some common cases for sanity.